msteams-platform/concepts/build-and-test/enable-camera-image-capabilities.md

```diff
@@ -0,0 +1,121 @@
+
+## Error handling
+
+You should understand the API response error codes and handle them appropriately. Below is a list of error codes that may be returned by the platform:
+
+|Error code |  Error Name     | Condition|
+| --- | --- | --- |
+| **100** | NOT_SUPPORTED_ON_PLATFORM | API not supported in the current platform.|
+| **404** | FILE_NOT_FOUND | The file specified was not found on the given location.|
+| **500** | INTERNAL_ERROR | Internal error encountered while performing the required operation.|
+| **1000** | PERMISSION_DENIED |Permissions denied by the user.|
+| **2000** |NETWORK_ERROR | Network issue.|
+| **3000** | NO_HW_SUPPORT | Underlying hardware doesn't support the capability.|
+| **4000**| INVALID_ARGUMENTS | One or more arguments is invalid.|
+| **5000** | UNAUTHORIZED_USER_OPERATION | User is not authorized to complete this operation.|
+| **6000** |INSUFFICIENT_RESOURCES | The operation couldn't be completed due to insufficient resources.|
+|**7000** | THROTTLE | The platform throttled the request because the API was invoked too frequently.|
+|  **8000** | USER_ABORT |User aborted the operation.|
+| **9000**| OLD_PLATFORM | The platform code is outdated and doesn't implement this API.|
+| **10000**| SIZE_EXCEEDED |  The return value is too big and has exceeded the platform size boundaries.|
+
+## Sample code snippets
+
+**Calling `selectMedia` API from a web app**
+
+```javascript
+let imageProp: ImageProps = {
+    sources: [SelectImageSource.Camera, SelectImageSource.Gallery],
+    startMode: SelectImageMode.Photo,
+    ink: false,
+    cameraSwitcher: false,
+    textSticker: false,
+    enableFilter: true,
+};
+let mediaInput: MediaInputs = {
+    mediaType: MediaType.Image,
+    maxMediaCount: 10,
+    imageProps: imageProp
+};
+microsoftTeams.selectMedia(mediaInput, (error: microsoftTeams.SdkError, attachments: microsoftTeams.Media[]) => {
+          if (error) {
+            if (error.message) {
+              alert(" ErrorCode: " + error.errorCode + error.message);
+            } else {
+              alert(" ErrorCode: " + error.errorCode);
+            }
+        }
+    if (attachments) {
+let y = attachments[0];
+          img.src = ("data:" + y.mimeType + ";base64," + y.preview);
+    }
+});
+```
+
+**Calling `getMedia` API from a web app**
+
+```javascript
+let media: microsoftTeams.Media = attachments[0]     
+media.getMedia((error: microsoftTeams.SdkError, blob: Blob) => {
+        if (blob) {
+          if (blob.type.includes("image")) {
+          img.src = (URL.createObjectURL(blob));
+          }
+      }
+      if (error) {
+          if (error.message) {
+            alert(" ErrorCode: " + error.errorCode + error.message);
+          } else {
+            alert(" ErrorCode: " + error.errorCode);
+          }
+      }
+      });
```

Comment:
Formatting of the closing bracket has unnecessary spaces/tabs in the front of it. Consider removing the tab/spaces entirely.